Add extension - Shopify Developer Changelog

sandypockets opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

Adding a new extension that reads an RSS feed from Shopify's Developer Changelog on [shopify.dev/changelog](https://shopify.dev/changelog), and displays it to the user as markdown. 

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

![image](https://user-images.githubusercontent.com/41454557/157152523-a63ac75f-2848-44ee-abd6-0795fcbebb12.png)

![image](https://user-images.githubusercontent.com/41454557/157152570-e3873c64-e27f-438e-b5c8-cb7c220b6830.png)

![image](https://user-images.githubusercontent.com/41454557/157152583-a40d399b-52c2-4fca-bf45-4564f910bd23.png)

![image](https://user-images.githubusercontent.com/41454557/157152597-6935a847-9d67-45e7-acfa-a84288e5d862.png)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `media` folder

fe9lix reviewed (commented):
Hey @sandypockets, thanks for your contribution! Some suggestions/questions:

- Would it make sense to use the current subtitle as title? That way, users could search for any strings in the title. You could add the category in the subtitle instead and maybe add an additional keyword for it? (keywords are also searchable)

- Is there a way to format the code highlighted blocks in the markdown? I've tested one detail view and got bold title formatting for the block that was code highlighted on the website. Not sure if that would require some processing of the received markdown? (We do support most CommonMark, so it could be shown as code block – without syntax highlighting, though).

sandypockets commented:
Hey @fe9lix thanks for the review and feedback! I have implemented your suggestions so that the: 

- Subtitle is now the title
- Content is properly parsed for CommonMark
- Screenshots have been updated

I also refactored a bit to make things easier to follow and maintain.

fe9lix commented:
Sounds good! 👍
